Sources/Bauletto/Bauletto.swift

```diff
@@ -211,16 +210,14 @@ public class Bauletto {
     
     /// Getting the key window of the current app.
     /// Returns 'UIWindow'.
-    fileprivate let keyWindow: UIWindow? = {
-        if #available(iOS 13.0, *) {
-            return UIApplication.shared.connectedScenes
-                .first { $0.activationState == .foregroundActive }
-                .map { $0 as? UIWindowScene }
-                .map { $0?.windows.first } ?? UIApplication.shared.delegate?.window ?? nil
+    private var keyWindow: UIWindow? {
+        if #available(iOS 13.0, *),
+           let scene = UIApplication.shared.connectedScenes.first(where: { $0.activationState == .foregroundActive }) as? UIWindowScene,
+           let window = scene.windows.first(where: { $0.isKeyWindow }) {
+            return window
```

Comment:
That is more readable, thanks! Can you please run a `ctrl + i` to indent it correctly? Thanks :)

Reply:
It's indented, it might look weird because it's an `if` with multiple lines.

Reply:
Ohhh I see, also the let in the if are indented. Great, will merge now 😉